<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeBootImage</class>
 <widget class="QDialog" name="MakeBootImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Boot Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/res/fw.png</normaloff>:/icons/res/fw.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Kernel</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="kernelFileName"/>
          </item>
          <item>
           <widget class="QToolButton" name="kernelSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ramdisk</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="ramdiskFileName"/>
          </item>
          <item>
           <widget class="QToolButton" name="ramdiskSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>DeviceTree</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="deviceTreeFileName"/>
          </item>
          <item>
           <widget class="QToolButton" name="deviceTreeSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Second Image</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="secondFileName"/>
          </item>
          <item>
           <widget class="QToolButton" name="secondImageSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Advanced Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Page Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pageSize"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Base Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="baseAddress"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Kernel Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kernelOffset"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ramdisk Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ramdiskOffset"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tag Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tagOffset"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="cmdLineGroup">
     <property name="title">
      <string>Command Line</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="commandLine"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Output Boot Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QToolButton" name="outputFileSelect">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputFileName"/>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="makeButton">
          <property name="text">
           <string>Make Boot Image</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/icons/res/reactor.png</normaloff>:/icons/res/reactor.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
